distributions/Dist_hk1764.py

```diff
@@ -0,0 +1,47 @@
+
+import numpy as np
+from base_distribution import BaseDistribution
+
+class Dist_hk1764(BaseDistribution):
+	def __init__(self):
+		self.f_max = 1
+		self.x_min = -1
+		self.x_max = 1
+
+
+	def pdf(self, x):
+		"""This is your PDF"""
+		return np.abs((x**2-x**3)/x)
+
+	def mean(self):
+		"""This is the mean of the PDF"""
+		return -.41
```

Comment:
Understood, I'll change it asap. 

Best,
Hark Kanwal

> On Apr 10, 2016, at 21:16, Kyle Cranmer <notifications@github.com> wrote:
> 
> In distributions/Dist_hk1764.py:
> 
> > +from base_distribution import BaseDistribution
> > +
> > +class Dist_hk1764(BaseDistribution):
> > +	def __init__(self):
> > +		self.f_max = 1
> > +		self.x_min = -1
> > +		self.x_max = 1
> > +
> > +
> > +	def pdf(self, x):
> > +		"""This is your PDF"""
> > +		return np.abs((x**2-x**3)/x)
> > +
> > +	def mean(self):
> > +		"""This is the mean of the PDF"""
> > +		return -.41
> There's nothing wrong with the PDF. The issue is that the mean and std should not just be numbers, but should be based on equations you derived using calculus and the definition of the mean and standard deviation of a distribution.
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
>